Exercise6/exercise6/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import "./App.css";
+
+function App() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await axios.get("http://localhost:8000/api/get");
```

Comment:
Specify the URL as an environment variable in Docker Compose file and then read it here through process.env.VARIABLENAME